File: tests/whileloop/whileloop.vb
Imports System
Imports System.Diagnostics

Public Class whileloop
    Public Shared Sub Main()
        Dim timer As Stopwatch = New Stopwatch()
        timer.Start()
        whilelooptest()
        timer.Stop()
        Console.Write(timer.ElapsedMilliseconds.ToString())
    End Sub

    Shared Sub whilelooptest()
        Dim heavyNo As Long = 1000000000
        Dim i As Long = 0
        While i < heavyNo
            i += 1
        End While
    End Sub
End Class

File: tests/whileloop/whileloop.cs
using System;
using System.Diagnostics;

public class forloop
{
	public static void Main()
	{
	    Stopwatch timer = new Stopwatch();
        timer.Start();
        forlooptest();
		timer.Stop();
		Console.Write(timer.ElapsedMilliseconds.ToString());
	}
	
	static void forlooptest()
	{
		long heavyNo = 1000000000;
		long i = 0;
		while(i < heavyNo)
		{
			i++; 
		}
	}
}